test/Sentry.Tests/Protocol/DsnTests.cs

```diff
@@ -30,14 +30,14 @@ public void Ctor_SampleValidDsnWithSecret_CorrectlyConstructs()
     public void Ctor_NotUri_ThrowsUriFormatException()
     {
         var ex = Assert.Throws<UriFormatException>(() => Dsn.Parse("Not a URI"));
-        Assert.Equal("Invalid URI: The format of the URI could not be determined.", ex.Message);
+        ex.Message.Should().BeOneOf("net_uri_BadFormat", "Invalid URI: The format of the URI could not be determined.");
```

Comment:
These strings are the pre-resolved string resource names from .NET.  They come through on Android in the emulators used in CI, but not locally.  I couldn't figure out why - but it didn't seem important to fix so I just allow them.